docs/cn/overview/Getting-Started.md

```diff
@@ -35,16 +35,16 @@ priority: 1
 
 ## 下载 Alluxio
 
-首先，从[这里](http://www.alluxio.io/download)下载 Alluxio。选择为默认 Hadoop 版本构建的{{site.ALLUXIO_RELEASED_VERSION}}预编译版。
+首先，从[这里](http://www.alluxio.io/download)下载 Alluxio。选择为默认Hadoop版本构建的相应预编译版。
```

Comment:
`首先，从[这里](http://www.alluxio.io/download)下载 Alluxio。选择为默认Hadoop版本构建的相应预编译版。`
-> `首先，从[这里](http://www.alluxio.io/download)下载 Alluxio。`
The following line can be removed since we only have the default hadoop version in download page